test_contour_plot1D.js

```diff
@@ -0,0 +1,91 @@
+function test_square(h, eps, nlim) {
+
+  //(Pre)defined functions
+  const square = (x) => x ** 2;
+  const pow3 = (x) => x ** 3;
+  const allGroup = ["square", "pow3"];
+  const objectives = {"square": {obj: square, x_ini: 1.5, delta: 0.8}, "pow3": {obj: pow3, x_ini: 1.5, delta: 0.1}};
+
+  // Create basic constituent of the Visualization (Contour Windows and button)
+  const contourPlot = new ContourPlot(
+    d3.select('#svg1'),
+    [-2, 2],
+    600,
+    600,
+  );
+  var dropdownButtonFunction = d3.select("#dataviz_builtWithD3")
```

Comment:
```suggestion
  var selectFunctionDropdownButton = d3.select("#dataviz_builtWithD3")
```
Otherwise we think the variable is a function, but we refer to the button element here :)